src/events/messageCreate.js

```diff
@@ -31,10 +31,11 @@ async function messageCreateHandler(message) {
 
 			// * Check that the message isn't too long to be sent
 			if (lmaod.length < 2000) {
-				await message.reply({
+				const reply = await message.reply({
 					content: lmaod,
 					allowedMentions: { parse: [] }
 				});
+				await reply.suppressEmbeds()
```

Comment:
Iirc this takes in a bool to either suppress or un-suppress embeds, should probably add `true` here to be safe?